src/Testing/Concerns/MakesAssertions.php

```diff
@@ -165,6 +165,15 @@ public function assertEmittedTo($target, $value, ...$params)
 
         return $this;
     }
+    public function assertNotEmittedTo($target, $value, ...$params)
+    {
+        $this->assertEmitted($value, ...$params);
```

Comment:
my point was: If you want to assert that the event is not emitted to a particular component, the event itself needs to be emitted in the first place, but NOT to that component. 

otherwise, if the event is not even emitted, why do we bother checking the component that the event is or is not emitted to? 
in this case, we can just use `assertNotEmitted`. 

does that make sense now? 

I've also explained in the [discussion](https://github.com/livewire/livewire/discussions/5047#discussion-4155783) the cases where this might be useful, and I have updated the sample code too. so you may have a look, please. and let me know what you think

thanks for reviewing my pr

Reply:
@nuernbergerA looking at the current implementation of `assertEmittedTo` it checks also asserts `assertEmitted` before checking that it was emitted to that specific component.

I think @omaresmael might be right, it's better to make sure the event actually got fired before you check to make sure it's not going to that specific component. Otherwise there might be false positives, whereby the event isn't actually getting emitted at all.

Thoughts?

Reply:
@joshhanley laravel has some similar assertions.
lets take `Notification::assertNotSentTo([$user], SomeNotification::class);` as an example.
it checks if the given user didn't receive `SomeNotification` but will not check if it was sent.

I understand why you will have the assertion alongside the other to double check if it was not emitted to the component or to any other component.

Maybe we need a lazy version of this assertion that checks that it wasn't emitted to the component, no matter if it was emitted at all.

And this should also be documented well

Reply:
so, should we do the same thing for the `assertEmittedTo` 
just check if the event is emitted to the component without checking if it was emitted in the first place. 
or that is a different case
please let me know
thanks

Reply:
@omaresmael I wouldn't be changing the existing `assertEmittedTo` as that would be a breaking change. For the moment just leave this PR the way it is until we can come up with a decent API for what @nuernbergerA is suggesting 🙂

@nuernbergerA are you suggesting two assertions one for if the event was fired at all with the check on the specific component and one for just if the specific component didn't receive it (no check)?

Have you got an idea for naming them or would you make it the same name with a 3rd parameter like isLazy/includeEmitted/ or something like that?